db/db_impl_secondary.cc

```diff
@@ -59,11 +103,29 @@ Status DBImplSecondary::Recover(
     single_column_family_mode_ =
         versions_->GetColumnFamilySet()->NumberOfColumnFamilies() == 1;
 
-    s = FindAndRecoverLogFiles();
+    std::unordered_set<ColumnFamilyData*> cfds_changed;
+    s = FindAndRecoverLogFiles(&cfds_changed, &job_context);
```

Comment:
`cfds_changed` isn't really needed during `Recover`, right? is it simpler to just pass nullptr here? 
If you have plans to used it later might be good to add a comment here?

Reply:
It's not used here, but if we allow it to be nullptr, then we need to do the check in `FindAndRecoverLogFiles` and the functions called by it.